tests/common/reboot.py

```diff
@@ -277,6 +281,12 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
         if not ret:
             raise Exception('warmboot-finalizer service timeout on DUT {}'.format(hostname))
 
+    # Verify if the temporary file created in tmpfs is deleted after reboot, to determine a
+    # successful reboot
+    file_check = duthost.command('find /dev/shm/test_reboot', module_ignore_errors=True)['rc']
+    if not file_check:
+        raise Exception('DUT {} reboot did not finish successfully'.format(hostname))
```

Comment:
It might be better to reword this as "DUT {} did not reboot" or "DUT {} did not appear to reboot".

Reply:
Modified it. Thank you!